src/Microsoft.ML.PipelineInference/PipelinePattern.cs

```diff
@@ -17,20 +17,35 @@ namespace Microsoft.ML.Runtime.PipelineInference
     /// </summary>
     public sealed class PipelinePattern : IEquatable<PipelinePattern>
     {
+        /// <summary>
```

Comment:
>summary [](start = 13, length = 7)

I think Tom meant xml comments above properties, not class itself.
```
///<summary>
/// The metric value of the training dataset result (not always used or set).
///</summary>
public double TrainingMetricValue { get; }

This way you will get tooltip in VS if you hover over them

Reply:
Thank you, this is helpful. I wasn't sure what he meant by XML comments, so I looked for examples, and saw them used as function comments. I'll make this change.